Source/santad/DataLayer/WatchItemsTest.mm

```diff
@@ -53,7 +54,8 @@
 extern bool ParseConfig(NSDictionary *config,
                         std::vector<std::shared_ptr<WatchItemPolicy>> &policies, NSError **err);
 extern bool IsWatchItemNameValid(NSString *watch_item_name, NSError **err);
-extern bool ParseConfigSingleWatchItem(NSString *name, NSDictionary *watch_item,
+extern bool ParseConfigSingleWatchItem(NSString *name, std::string_view policy_version,
+                                       NSDictionary *watch_item,
```

Comment:
```suggestion
extern bool ParseConfigSingleWatchItem(NSString *name,
                                       std::string_view policy_version,
                                       NSDictionary *watch_item,
```

Reply:
Same here too - If I change this, `clang-format` changes it back :(